students/ZidanLuo/session09/donor_models.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env python
+import sys
+import os
+
+class Donor_Collection():
+    def __init__(self):
+        self.donor_dict = {"Fred Jones": Donor("Fred Jones", 500)}
+        self.sorted_dict = {"Fred Jones": Donor("Fred Jones", 500)}
```

Comment:
you never want the same data stored twice -- so either sort on demand, or sort whenever you add or remove a donor.